homeassistant/components/tautulli/strings.json

```diff
@@ -0,0 +1,31 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Tautulli",
+        "description": "To find your API key, open the Tautulli web interface and navigate to Settings and then to Web interface. The API key will be at the bottom of that page.\n\nExample of the URL: http://192.168.0.10:8181 with 8181 being the default port.",
```

Comment:
Should we escape/code-fence the URL to prevent the UI from creating a dead link?

Is it "Web interface" or "web interface"? Both variants are in this line.